defi/src/protocols/data3.ts

```diff
@@ -45007,7 +45007,7 @@ const data3: Protocol[] = [
     cmcId: null,
     category: "Yield Aggregator",
     chains: ["Solana"],
-    oracles: ["Pyth"], // https://nx-finance.gitbook.io/nx-finance-whitepaper/protocol-mechanism/risks#oracle-risks
+    oracles: ["Pyth", "Switchboard"], // https://nx-finance.gitbook.io/nx-finance-whitepaper/protocol-mechanism/risks#oracle-risks
```

Comment:
Integration post:

https://x.com/NX_Finance/status/1813574596526047395

Reply:
Please provide more proofs regarding this
Can only see Pyth
https://nx-finance.gitbook.io/nx-finance-whitepaper/audits-and-security/oracle

Reply:
Removing NXFinance until reply from their team